modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -1098,7 +1098,7 @@ When defining an output of type XML, there are a few optional parameters you can
 [cols="2,1,1,2", options="header"]
 |===
 |Parameter |Type |Default|Description
-| `escapeCR` | Boolean | `false` | Whether to escape carriage returns (CR) chars or not. Valid Options: `true` or `false`
+//| `escapeCR` | Boolean | `false` | Whether to escape carriage returns (CR) chars or not. Valid Options: `true` or `false`
```

Comment:
Not a biggie but your descriptions are different - one is singular and one is plural. The second one should also be singular. "returns (CR) chars sounds funny,